modules/k3s/src/test/java/org/testcontainers/k3s/Fabric8K3sContainerTest.java

```diff
@@ -26,9 +26,18 @@ public class Fabric8K3sContainerTest {
 
     @Test
     public void shouldStartAndHaveListableNode() {
+        runK3s(DockerImageName.parse("rancher/k3s:v1.21.3-k3s1"));
+    }
+
+    @Test
+    public void shouldStartAndHaveListableNode_backwardsCompat() {
+        runK3s(DockerImageName.parse("rancher/k3s:v1.17.17-k3s1"));
+    }
+
+    private void runK3s(DockerImageName k3sDockerImage) {
         try (
             // starting_k3s {
-            K3sContainer k3s = new K3sContainer(DockerImageName.parse("rancher/k3s:v1.21.3-k3s1"))
+            K3sContainer k3s = new K3sContainer(k3sDockerImage)
```

Comment:
This change will break [the documentation](https://deploy-preview-7677--testcontainers.netlify.app/modules/k3s/#usage-example). We prefer to specify the image in this case. I think it will be totally fine to keep this test as is and the other one can be updated instead.